### ISSUE TITLE
“Unfortunately, your wallet couldn't be recovered.”

### ISSUE THREAD

**Issue by da2x** [mikel2000/batify#28]
Can I not backup a wallet from the Brave browser and recover the backup in BATify?

```
BATify: ledger.recoverWallet: ERROR: recover wallet failed: passphrase is invalid. background.js:3816:10
BATify: cryptoUtils.passphraseToBytes: ERROR: failed: Error: invalid word: {redacted} background.js:3816:10
BATify: ledger.recoverWallet: wallet couldn't be recovered: undefined background.js:3816:10
```

I {redacted} a three letter word in the fifth position of my wallet recovery string. There are no special characters or oddities in the recovery string.

```
BATify version: 1.0.4
Browser: Firefox 61.0
OS: Fedora 28
Brave (wallet export): 0.23.31
```

**Comment by mikel2000** [comment]
Normally this should work. If I remember correctly, I tried this some time ago successfully. How many words has your passphrase got?

**Comment by da2x** [comment]
23 words.

**Comment by mikel2000** [comment]
That's strange. BATify expects a passphrase with 16 words. Seed length is 32 bytes, 2 bytes per word. Did you try to recover your Brave wallet with this passphrase?

**Comment by da2x** [comment]
I’m trying to restore a wallet backup from Brave in BATify.

On one computer, my Brave wallet key is 23 words. On another computer I have, it’s 24 words.

**Comment by mikel2000** [comment]
> I’m trying to restore a wallet backup from Brave in BATify.

Sure, I understand. Could you please try to restore your Brave wallet with the Brave generated passphrase?

**Comment by mikel2000** [comment]
I checked the brave client code in the meantime. It seems to be, that they are using now another tool for passphrase generating (before they used niceware). So, unfortunately, BATify and Brave are no longer compatibel at this point.

**Comment by da2x** [comment]
Could you inude some documentation in the client saying it only works with BATify and that it isn't compatible with wallets coming from Brave?

**Comment by mikel2000** [comment]
Will be solved in #29.
